app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -90,203 +88,86 @@ For safeguarding reasons, schools will assess your suitability to teach from a r
 
-[Learn more about the new service](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers/a-fairer-approach-to-awarding-qts-to-overseas-teachers--2#). 
+* [International English Language Testing System (IELTS)](https://www.ielts.org/)
+* [Test of English as a Foreign Language (TOEFL)](https://www.ets.org/toefl)
+* [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)
 
-### Visas and immigration
+### Assessment only QTS
+If you find you do not meet the eligibility criteria to apply for QTS, you could consider assessment only QTS. This means you can work in an English school and, alongside your job, apply to an English teacher training provider who will assess if you meet the standards for English QTS.
 
-Most Ukrainian refugees will not need to be sponsored for a skilled worker or student visa. [Find out more about visa schemes for Ukrainian refugees, allowing them to live, work and study in the UK](#check-the-status-of-your-visa).
+Check with [assessment only training providers](/train-to-be-a-teacher/assessment-only-route-to-qts) for more information about eligibility.
 
-### Qualifications
+### Other teaching job options
 
-Schools should assess the qualifications and experience of Ukrainian teachers in the same way as other overseas applicants, bearing in mind the exemption provided by the [4 year rule](https://www.gov.uk/guidance/recruit-teachers-from-overseas#employing-overseas-teachers-without-qts-the-4-year-rule).
+[Supply teachers](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher) fill in when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school.  
 
-UK ENIC can provide a [statement of comparability](https://www.enic.org.uk/Qualifications/SOC/Default.aspx) to applicants showing how their academic and professional qualifications compare to the English equivalents.
+[Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
 
-### Alternative roles in school
+You could also consider [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) and education support roles. It can be helpful to gain experience of the English education system before you apply for teaching jobs. [Teaching Vacancies](https://teaching-vacancies.service.gov.uk/) lists these jobs. 
 
-DfE encourages schools to employ Ukrainian teachers and prospective teachers as teaching assistants if a qualified teacher role isn't available. Ukrainians can then be supported to gain QTS via [assessment only QTS](#assessment-only-qts). 
+For a short-term unpaid placement, you can [request school experience in a primary or secondary school in England](https://schoolexperience.education.gov.uk/). 
```

Comment:
@laurahermionetennant92 the service is called Get school experience so think it would be better to change request to get here.